src/assisted_test_infra/test_infra/helper_classes/infra_env.py

```diff
@@ -100,6 +100,15 @@ def update_host(
             disks_skip_formatting=disks_skip_formatting,
         )
 
+    def update_host_installer_args(self, host_id: str):
+        if not self._config.host_installer_args:
```

Comment:
Do you think that update_host_installer_args may be called during cluster lifetime ? i mean allow to user update host_installer_args ?  the way its written no way to update on the fly